doc/Overview/Mvux/Advanced/InspectGeneratedCode.md

```diff
@@ -0,0 +1,26 @@
+---
+uid: Overview.Mvux.Advanced.InspectGeneratedCode
+---
+
+# Inspecting the generated code
+
+MVUX makes extensive use of code generation. It's used to generate Bindable Proxy Models and entities and commands.  
+Inspecting the generated code can give you a lot of insight into how the Bindable Proxies work under the hood and can also improve your knowledge about MVUX operates behind the scenes and familiarize yourself with its internals.
+
+Viewing the generated code can be achieved in several ways:
+
+1. Placing the cursor on the class name and hitting <kbd>F12</kbd>:
+
+    ![Demonstration of hitting F12 in Visual Studio to see generated code](../Assets/InspectingGeneratedCode-1.gif)
+
+1. Hitting <kbd>Ctrl</kbd>+<kbd>T</kbd> and typing in the Bindable type name:
+
+    ![Demonstration of hitting Control T in Visual Studio to see generated code](../Assets/InspectingGeneratedCode-2.gif)
+
+1. Another way that can also be used to inspect all code generated by MVUX and even other code generators, is by navigating to the project's analyzers.
```

Comment:
Does this need "Another way that..." given that this is a list of ways of "achieving" the viewing of the generated code?

Reply:
```suggestion
1. By navigating to the project's analyzers:

    This method can also be used to inspect all code generated by MVUX and even other code generators.
```